examples/vpc_service_control/onprem_project/terraform.tfvars

```diff
@@ -0,0 +1,7 @@
+project_id = ""
```

Comment:
Please remove all terraform.tfvars.
If you want to provide an example terraform.tfvars create a terraform.tfvars.example file and add placeholder to in variables that need to be configured by the user. Then instruct in the README to rename the file to terraform.tfvars before running the example.

Reply:
I think this limits the usefulness of the example.  Not every user has experience with Terraform, nor do they realize a terraform.tfvars file is necessary.  I'll remove the tfvars files if that's "the policy," but I don't think this helps people who are brand new to Terraform.

Reply:
Yeah it;s kind of the "policy". it's actually best practices for CFT as the user should only upload terraform.tfvars files in their own repos that use the modules. I agree that a user may be unfamiliar with TF and not realize that it's necessary, but I think the alternative of using a terraform.tfvars.example is good compromise that shows the users that terraform.tfvars is necessary while not having to create a terraform.tfvars from scratch.